docs/source/api/v2/deliveryservice_request_comments.rst

```diff
@@ -0,0 +1,318 @@
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-deliveryservice_request_comments:
+
+************************************
+``deliveryservice_request_comments``
+************************************
+
+``GET``
+=======
+Gets delivery service request comments.
+
+:Auth. Required: Yes
+:Roles Required: None
+:Response Type:  Array
+
+Request Structure
+-----------------
+
+.. table:: Request Query Parameters
+
+	+--------------------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Name                     | Required | Description                                                                                                                                         |
+	+==========================+==========+=====================================================================================================================================================+
+	| author                   | no       | Filter for :ref:`Delivery Service Request <ds_requests>` comments submitted by the user identified by this username                                 |
+	+--------------------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+	| authorId                 | no       | Filter for :ref:`Delivery Service Request <ds_requests>` comments submitted by the user identified by this integral, unique identifier              |
+	+--------------------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+	| deliveryServiceRequestId | no       | Filter for :ref:`Delivery Service Request <ds_requests>` comments submitted for the delivery service identified by this integral, unique identifier |
+	+--------------------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+	| id                       | no       | Filter for the :ref:`Delivery Service Request <ds_requests>` comment identified by this integral, unique identifier                                 |
+	+--------------------------+----------+-----------------------------------------------------------------------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/2.0/deliveryservice_request_comments HTTP/1.1
+	User-Agent: python-requests/2.22.0
+	Accept-Encoding: gzip, deflate
+	Accept: */*
+	Connection: keep-alive
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+
```

Comment:
Response field descriptions?

Reply:
Response field definitions added in 7378db3677